desktop/app/console-helper.js

```diff
@@ -42,8 +42,8 @@ function tee (...writeFns) {
   return t => writeFns.forEach(w => w(t))
 }
 
-const stdErrWriter = t => process.stderr.write(t)
-const stdOutWriter = t => process.stdout.write(t)
+const stdErrWriter = t => process.platform !== 'win32' && process.stderr.write(t)
+const stdOutWriter = t => process.platform !== 'win32' && process.stdout.write(t)
```

Comment:
This means we won't see stdout logs even if we run it from the cmd line in windows. I'm okay with that for now, but maybe later we can see if we can write first, and then decide to log to stdout